cranelift/codegen/src/isa/riscv64/inst/emit.rs

```diff
@@ -1382,11 +1387,12 @@ impl MachInstEmit for Inst {
                 sink.bind_label(label_true, &mut state.ctrl_plane);
                 Inst::load_imm12(rd, Imm12::TRUE).emit(&[], sink, emit_info, state);
                 Inst::Jal {
-                    dest: BranchTarget::offset(Inst::INSTRUCTION_SIZE * 2),
```

Comment:
As a follow-up, should `Inst::INSTRUCTION_SIZE` be deleted if the instruction size is going to become variable?

Reply:
I think we should rename it to something like `UNCOMPRESSED_INSTRUCTION_SIZE`. It's still used in a couple of places for example when checking if we need to emit an island.

It is also used when calculating `br_table` offsets, which are going to be forced to emit uncompressed instructions.